### PR TITLE
fix: error on return + apply code styles

### PR DESCRIPTION
**Pull request by stepcha**
- api_call method return, always was expecting an error key (now is optional)
- apply black, isort and some suggestions from flake8
- remove unused import of open-cv lib

### CODE REVIEW

File: searchads_api/api.py [hunk 1]
```diff
@@ -1283,24 +1219,24 @@ def get_product_page_locales(self, adamId, productPageId, expand, languages, lan
...
-                            params=data, method="GET")
+        res = self.api_call(
+            f"apps/{adamId}/product-pages/{productPageId}/locale-details",
+            params=data,
+            method="GET",
+        )
         return res
 
-    def get_product_page_locales(self, countriesOrRegions):
+    def get_supported_languages(self, countriesOrRegions):
```
Comment (stepcha) [inline searchads_api/api.py:1231]: @alkattan here is the real change

File: searchads_api/api.py [hunk 2]
```diff
@@ -1264,11 +1197,14 @@ def get_product_page_by_identifier(self, adamId, productPageId):
...
         productPageId: int,  (Required) A unique string to identify a product page on App Store Connect. For example, 45812c9b-c296-43d3-c6a0-c5a02f74bf6e.
         """
-        res = self.api_call(f"apps/{adamId}/product-pages/{productPageId}",
-                            method="GET")
+        res = self.api_call(
+            f"apps/{adamId}/product-pages/{productPageId}", method="GET"
+        )
         return res
 
-    def get_product_page_locales(self, adamId, productPageId, expand, languages, languageCodes, deviceClasses):
```
Comment (stepcha) [inline searchads_api/api.py:1272]: @alkattan duplicate function

File: searchads_api/api.py [hunk 3]
```diff
@@ -1283,24 +1219,24 @@ def get_product_page_locales(self, adamId, productPageId, expand, languages, lan
...
-        res = self.api_call(f"apps/{adamId}/product-pages/{productPageId}/locale-details",
-                            params=data, method="GET")
+        res = self.api_call(
+            f"apps/{adamId}/product-pages/{productPageId}/locale-details",
+            params=data,
+            method="GET",
+        )
         return res
 
-    def get_product_page_locales(self, countriesOrRegions):
```
Comment (stepcha) [inline searchads_api/api.py:1293]: @alkattan duplicate function

**Review by alkattan — APPROVED** [review]
approved

**Pull request merged**